app/models/application_answer.rb

```diff
@@ -0,0 +1,4 @@
+class ApplicationAnswer < ApplicationRecord
```

Comment:
`Application`* is a bit of a confusing name to use for objects as we see the string `Application` in lots of code

Reply:
Yeah, not sure there's a good answer here. Maybe "SessionApplication"? Of course, "session" is overused as well.